Soomla/Assets/Plugins/Soomla/Profile/IOSImpl/ProfileEventPusherIOS.cs

```diff
@@ -226,21 +228,25 @@ public class ProfileEventPusherIOS : Soomla.Profile.ProfileEvents.ProfileEventPu
 			soomlaProfile_PushEventGetScoresFailed(ev.Provider.ToString(), ev.From.toJSONObject().ToString(), ev.ErrorDescription, ev.FromStart, ev.Payload);
 		}
 
-		protected override void _pushEventReportScoreStarted(ReportScoreStartedEvent ev) {
+		protected override void _pushEventSubmitScoreStarted(SubmitScoreStartedEvent ev) {
 			if (SoomlaProfile.IsProviderNativelyImplemented(ev.Provider)) return;
 			soomlaProfile_PushEventReportScoreStarted(ev.Provider.ToString(), ev.Destination.toJSONObject().ToString(), ev.Payload);
 		}
 
-		protected override void _pushEventReportScoreFinished(ReportScoreFinishedEvent ev) {
+		protected override void _pushEventSubmitScoreFinished(SubmitScoreFinishedEvent ev) {
 			if (SoomlaProfile.IsProviderNativelyImplemented(ev.Provider)) return;
 			soomlaProfile_PushEventReportScoreFinished(ev.Provider.ToString(), ev.Destination.toJSONObject().ToString(), ev.Score.toJSONObject().ToString(), ev.Payload);
 		}
 
-		protected override void _pushEventReportScoreFailed(ReportScoreFailedEvent ev) {
+		protected override void _pushEventSubmitScoreFailed(SubmitScoreFailedEvent ev) {
 			if (SoomlaProfile.IsProviderNativelyImplemented(ev.Provider)) return;
 			soomlaProfile_PushEventReportScoreFailed(ev.Provider.ToString(), ev.Destination.toJSONObject().ToString(), ev.ErrorDescription, ev.Payload);
 		}
 
+		protected override void _pushEventShowLeaderboards(ShowLeaderboardsEvent ev) {
+			if (SoomlaProfile.IsProviderNativelyImplemented(ev.Provider)) return;
```

Comment:
@EugeneButusov,
1. Never put this on 1 line.
2. Always use {}
3. When you copy bad code, you become an author of this code.